components-comparison.html

```diff
@@ -0,0 +1,1484 @@
+                        </details>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">ad</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4 no"></td>
+                    <td class="c5"><code>&lt;ad&gt;</code></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">payment</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4 no"></td>
+                    <td class="c5"><code>&lt;inline-payment-panel&gt;</code></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">comments</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4 no"></td>
+                    <td class="c5">
+                        <details>
+                            <summary>
+                                <code>&lt;comment-list&gt;</code>
+                            </summary>
+                            <code>&lt;comment-list&gt;</code>, <code>&lt;comment-detail&gt;</code>
+                        </details>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">like</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4 no"></td>
+                    <td class="c5"><code>&lt;like&gt;</code></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">3d-model</td>
+                    <td class="c2 partial"><code>&lt;model&gt;</code> (<abbr title="Work in Progress">WIP</abbr>)</td>
+                    <td class="c3 no"></td>
+                    <td class="c4 no"></td>
+                    <td class="c5"><code>&lt;modelviewer&gt;</code></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">rating</td>
+                    <td class="c2 no"></td>
+                    <td class="c3"><code>&lt;rating&gt;</code></td>
+                    <td class="c4 no"></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">marquee</td>
+                    <td class="c2 no"></td>
```

Comment:
There's https://html.spec.whatwg.org/multipage/rendering.html#the-marquee-element-2

Reply:
Thanks @xfq . My brain omitted it. I remember  using it :) Changed at 48b71421bad355590535d06c68b4187e938be234